contrib/vagrant/start.sh

```diff
@@ -306,8 +306,7 @@ function write_cilium_cfg() {
 
     cilium_options="\
       --debug --pprof --enable-hubble --hubble-listen-address :4244 --enable-k8s-event-handover \
-      --k8s-require-ipv4-pod-cidr --enable-bandwidth-manager --kube-proxy-replacement=disabled \
-      --enable-remote-node-identity"
```

Comment:
Consider moving this commit to the beginning of the git history. Using a flag which is removed a previous commit might break `git bisect`.